src/main/java/de/tum/in/www1/artemis/domain/hestia/TestwiseCoverageReportEntry.java

```diff
@@ -0,0 +1,70 @@
+package de.tum.in.www1.artemis.domain.hestia;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+import de.tum.in.www1.artemis.domain.ProgrammingExerciseTestCase;
+
+/**
+ * A single entry from testwise coverage report by file path and consecutive executed code block.
+ * A block is represented by the start line and the length (i.e. number of lines) of the block.
+ */
+@Entity
+@Table(name = "testwise_coverage_report_entry")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class TestwiseCoverageReportEntry extends DomainObject {
+
+    @ManyToOne
+    @JsonIgnoreProperties("testwiseCoverageEntries")
+    private CoverageFileReport fileReport;
+
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
I think EAGER is default here. We normally leave it out for ManyToOne, because all non default EAGER values need to be documented.